### PR TITLE
Add MBS form 0123 0173

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

File: data/en/mbs_0123.json [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "title": "",
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                "question": "Sales on own account and turnover from other activities",
```
Comment (shahi645) [inline data/en/mbs_0123.json]: @MIslam92 Is the front page supposed to match the spec? still missing some content

Comment (nealedj) [reply data/en/mbs_0123.json]: I think there were changes to the front page that may not have been picked up. @shahi645 as Sam has moved team would you mind picking this up please?

Comment (shahi645) [reply data/en/mbs_0123.json]: Yeah i can pick this up, speaking to him this morning though, i think he will be allowed to finish this

File: data/en/mbs_0123.json [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+                "id": "summary-group",
+                "title": "Summary",
+                "blocks": [{
+                    "type": "Summary",
+                    "id": "summary"
+                }]
+            }]
+        }
+    ]
+}
```
Comment (shahi645) [inline data/en/mbs_0123.json:356]: Happy with the schema apart from front page

File: data/en/mbs_0173.json [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Sales on own account and turnover from other activities",
```
Comment (shahi645) [inline data/en/mbs_0173.json]: This page differs from spec also, need clarification

File: data/en/mbs_0123.json (hunk 1 above)
Comment (charlotte-hirst) [inline data/en/mbs_0123.json]: is this not missing commission and fees question

File: data/en/mbs_0123.json (hunk 2 above)
Comment (MIslam92) [reply data/en/mbs_0123.json:356]: @shahi645 If your referring to 'Commission and fees' that should not be there anymore anyway, spec needs to be updated and same on the other survey as well

File: data/en/mbs_0173.json [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+                                    "You can provide informed estimates if actual figures aren’t available.",
+                                    "We will treat your data securely and confidentially."
+                                ]
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
```
Comment (charlotte-hirst) [inline data/en/mbs_0173.json:39]: missing commission and fees and employment section

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

**Comment by shahi645** [comment]
LGTM, only a couple of queries around intro pages

File: data/en/mbs_0123.json (hunk 2 above)
Comment (shahi645) [reply data/en/mbs_0123.json:356]: Is this for definite?

File: data/en/mbs_0123.json (hunk 1 above)
Comment (MIslam92) [reply data/en/mbs_0123.json]: I thought this was meant to be removed as it only shows an example on the dropdown?

File: data/en/mbs_0173.json (hunk 4 above)
Comment (MIslam92) [reply data/en/mbs_0173.json:39]: Same as previous comment

**Review by shahi645 — APPROVED** [review]
👍

**Review by charlotte-hirst — APPROVED** [review]

**Pull request merged**